leapp/reporting/__init__.py

```diff
@@ -326,6 +326,11 @@ def _create_report_object(entries):
     _sanitize_entries(entries)
     for entry in entries:
         entry.apply(report)
+    if Groups.INHIBITOR in report.get('groups', []):
+        # Keep the orig inhivitor functionality until we drop it completely
```

Comment:
```suggestion
        # Keep the original inhibitor functionality until we drop it completely
```
Idea: maybe the comment could provide insights how the inhibitor detection works  
```suggestion
        # Before removing Flags, the original inhibitor detection worked by checking the `flags` field; keep the `flags` field until we drop Flags completely
```

Reply:
Good point!